app/uk/gov/hmrc/hmrcemailrenderer/templates/mandate/mandateAgentActivates.scala.html

```diff
@@ -0,0 +1,8 @@
+@(params: Map[String, Any])
+
+@uk.gov.hmrc.hmrcemailrenderer.templates.helpers.html.template_main(params, params("service").toString) {
+
```

Comment:
We do not know whom we are sending it out to...and as it's a general information/notification, the name is not very important